UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEditor;
```

Comment:
I did not see this in my initial review. You can include UnityEditor libs in runtime script if you put them in compile time definitions. So you can add:
```
#if UNITY_EDITOR
using UnityEditor;
#endif
```
And it will remove this line at compile time. Anything code relying on this lib will also need to be placed in these definitions. But the code won't work in a build obviously as the code is removed at compile time.